src/main/resources/entity-types/users/drv_custom_field_details.json5

```diff
@@ -0,0 +1,344 @@
+{
+  id: '0b70ae1e-a87a-4621-a2d8-143973821e39',
+  name: 'drv_custom_field_details.json5',
```

Comment:
json5 shouldn't be in the name here (also maybe add user, eg drv_user_custom_field_details)?